app/components/ContactUsSection/ContactUsSection.tsx

```diff
@@ -9,31 +9,35 @@ type TProps = {
   isMainPage?: boolean;
 };
 
-export const ContactUsSection: FunctionComponent<TProps> = ({ isMainPage }) => {
-  const buttonBgColor = isMainPage ? "bg-accent" : "bg-black";
-  const containerBgColor = isMainPage ? "bg-black" : "bg-accent";
-  const containerClassName = cn(
-    "py-[60px] px-4 flex flex-col items-center gap-[60px] rounded-2xl",
-    containerBgColor,
-  );
-  const buttonClassName = cn(
-    buttonBgColor,
-    "hover:bg-white hover:text-accent transition-all",
-  );
+export const ContactUsSection = forwardRef<HTMLDivElement, TProps>(
+  ({ isMainPage }, ref) => {
+    const buttonBgColor = isMainPage ? "bg-accent" : "bg-black";
+    const containerBgColor = isMainPage ? "bg-black" : "bg-accent";
+    const containerClassName = cn(
+      "py-[60px] px-4 flex flex-col items-center gap-[60px] rounded-2xl",
+      containerBgColor,
+    );
+    const buttonClassName = cn(
+      buttonBgColor,
+      "hover:bg-white hover:text-accent transition-all",
+    );
```

Comment:
```suggestion
    const containerClassName = cn(
      "py-[60px] px-4 flex flex-col items-center gap-[60px] rounded-2xl",
      isMainPage ? "bg-black" : "bg-accent",
    );
    const buttonClassName = cn(
      isMainPage ? "bg-accent" : "bg-black",
      "hover:bg-white hover:text-accent transition-all",
    );
```

Let's remove these extra variables they are not needed as they are used only at a single place